src/main/java/mate/academy/dao/MovieDao.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.dao;
+
+import java.util.Optional;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Movie;
+
+public interface MovieDao {
+    Movie add(Movie movie) throws DataProcessingException;
+
+    Optional<Movie> get(Long id) throws DataProcessingException;

```

Comment:
why do we need here "throws DataProcessingException"?

Reply:
we don't need, it's my fault, fixed